packages/reactive-element/src/reactive-element.ts

```diff
@@ -1421,12 +1464,21 @@ export abstract class ReactiveElement
         .elementProperties;
       if (elementProperties.size > 0) {
         for (const [p, options] of elementProperties) {
+          const {wrapped, defaultValue} = options;
+          if (wrapped !== true && defaultValue === undefined) {
+            continue;
+          }
+          const value = this[p as keyof this];
+          // Ensure any initially set value is seen as a change.
```

Comment:
Do we want to do this for default values? It kind of makes sense to me to consider the initial default value _not_ changed.

Or maybe that's what this does? Might be clearer if the comments are inside the conditions? If so, maybe mention that in the jsdocs for the defaultValue option too.

Reply:
Made the comments clearer. Properties with defaultValue set _are_ currently included in initial changes.

We could choose to change this but the number of different ways properties can be configured makes this really hard to manage. (In fact there are some issues with this currently: `accessor` defined properties are only handled specially with the decorator and not in JS.)